packages/commerce-server/src/get-active-merchant-coupon.ts

```diff
@@ -14,7 +14,9 @@ export async function getActiveMerchantCoupon({
 
   let activeMerchantCoupon = null
 
-  const defaultCoupons = await getDefaultCoupon(productId)
+  const defaultCoupons = productId
+    ? await getDefaultCoupon([productId])
```

Comment:
Is getDefaultCoupon capable of accepting an array as an argument? [async getDefaultCoupon(productId?: string) {](https://github.com/skillrecordings/products/blob/d4c420f10094235a96a923ec2d3afc547276acd5/packages/database/src/prisma-api.ts#L644-L667)

Reply:
as part of this PR is is now